### PR TITLE
#731 Section Examples in Dynawo Modelica library with SMIB_StepPm

### PR DESCRIPTION
**Pull request by MarianneSaugierRTE**
First test case with graphical view and description. Ideas and comments are welcome @cossartque @RomainLosseauRTE

### CODE REVIEW

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIB_StepPm.mo [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  line1.switchOffSignal2.value = false;
+  line2.switchOffSignal1.value = false;
+  line2.switchOffSignal2.value = false;
+  transformer.switchOffSignal1.value = false;
+  generatorSynchronous.switchOffSignal1.value = false;
+  generatorSynchronous.switchOffSignal2.value = false;
+  generatorSynchronous.switchOffSignal3.value = false;
+  annotation(
+    experiment(StartTime = 0, StopTime = 30, Tolerance = 0.0001, Interval = 0.015),
+    Documentation(info = "<html><head></head><body> This test case represent a synchronous machine connected to an infinite bus through a transformer and two lines in parallel. <div><br></div><div> The simulated event is a step variation on the generator  mechanical power Pm.</body></html>"));
```
Comment (RomainLosseauRTE) [inline dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIB_StepPm.mo]: represents

**Comment by RomainLosseauRTE** [comment]
> First test case with graphical view and description. Ideas and comments are welcome @cossartque @RomainLosseauRTE

This looks good to me, except maybe the generator view which could be a bit more symmetrical

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIB_StepPm.mo (hunk 1 above)
Comment (MarianneSaugierRTE) [reply dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIB_StepPm.mo]: fixed

File: dynawo/sources/Models/Modelica/Dynawo/Examples/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
```
Comment (adriguir) [inline dynawo/sources/Models/Modelica/Dynawo/Examples/CMakeLists.txt]: When adding new files to the OS repo, it is better to update the copyright to 2015-2020.

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIB_StepPm.mo [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model SMIB_StepPm "Synchronous machine infinite bus - Step on Pm"
```
Comment (adriguir) [inline dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIB_StepPm.mo]: This is an open question: most of our models in the Modelica library are without "_" but with something like SMIBStepPm? The only "_" are reserved for "_INIT" models at the moment. Do we want to keep to this coding style?

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/package.mo [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+package SMIB "Synchronous Machine Infinite Bus test cases"
+  extends Icons.Package;
+  annotation(
+    Documentation(info = "<html><head></head><body>The SMIB test case was built based on Kundur Power System Stability and Control book. It is is a well-known test case in the power system community. It is very often used to illustrate the transient or the small-signal stability of a synchronous machine. It is also used to demonstrate the behavior of a new regulation, such as a speed governor, a voltage regulation or a power-system stabilizer.</body></html>"));
```
Comment (adriguir) [inline dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/package.mo]: Perhaps add the reference to some pages in Kundur book

File: dynawo/sources/Models/Modelica/Dynawo/Examples/CMakeLists.txt (hunk 2 above)
Comment (MarianneSaugierRTE) [reply dynawo/sources/Models/Modelica/Dynawo/Examples/CMakeLists.txt]: sorry, I will correct, nice catch!

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIB_StepPm.mo (hunk 3 above)
Comment (MarianneSaugierRTE) [reply dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/SMIB_StepPm.mo]: you're right, I just named it after dynawo test case. I will rename it

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/package.mo (hunk 4 above)
Comment (MarianneSaugierRTE) [reply dynawo/sources/Models/Modelica/Dynawo/Examples/SMIB/package.mo]: example 13.1 added

**Pull request merged**